advanced_functionality/huggingface_byo_scripts_and_data/huggingface-custom-text-summarizer.ipynb

```diff
@@ -0,0 +1,2886 @@
+{
```

Comment:
<p>This doesn't work in some Studio containers due to the containers using outdated Pandas version (we are working to fix that but it is taking a long time as we need cooperation from other teams). To make it friendly across multiple kernels for now, can you download the file from S3 using boto3 and then use pandas to read in the downloaded file?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3095/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>